functions/index.js

```diff
@@ -1,12 +1,13 @@
 const functions = require("firebase-functions")
 const express = require('express')
 const cors = require('cors')
-const { getTasks, createTask } = require('./src/tasks')
+const { getTasks, createTask, getSingleTask } = require('./src/tasks')
 
 const app = express()
 app.use(cors())
 
 app.get('/tasks', getTasks)
 app.post('/tasks', createTask)
+app.get('/singletask', getSingleTask)
```

Comment:
Need to pass a param

Reply:
`app.get('/tasks/:taskId', getSingleTask)`